packages/tooling/fast-tooling/src/web-components/color-picker/color-picker.ts

```diff
@@ -7,11 +7,11 @@ import {
 } from "@microsoft/fast-colors";
 import { attr, DOM, observable } from "@microsoft/fast-element";
 import { isNullOrWhiteSpace } from "@microsoft/fast-web-utilities";
-import { FASTTextField } from "@microsoft/fast-components";
+import { fastTextField } from "@microsoft/fast-components";
+import { DesignSystem } from "@microsoft/fast-foundation";
 import { FormAssociatedColorPicker } from "./color-picker.form-associated";
 
-// Prevent tree shaking
-FASTTextField;
+DesignSystem.getOrCreate().register(fastTextField());
```

Comment:
My recommendation would be to remove this. Just make one call to getOrCreate and pass all the components to it in a central location.

Reply:
I can insist that if anyone uses these `experimental` components that they have to include all dependencies themselves, but as far as I'm aware it's possible that before release we may not have a way of having implicit registrations occurring. This is not ergonomic for someone who's only taking a dependency on tooling and would expect to only register, for example, the color picker.

Imagining for a moment that the `<Form />` relied on the FAST components and aside from importing and using it, the app author also had to register all of its web component dependencies separately. I would in this case have preferred to wait on adoption until we had an ergonomic solution. Fortunately only `experimental` components are using the FAST components at this time.